### DISCUSSION TITLE
[Jetbrains Plugin] I can't close the copilot panel

### DISCUSSION THREAD

**Discussion by duxtinto** [community/community#7297] [Copilot Conversations] [answered]
Hi! Good afternoon!

I have just installed the last version of the jetbrains plugin on PhpStorm 2021.2.3, and after opening the github copilot panel, I don't know how to close it :sweat_smile: , as there is no minimization icon on it, as the rest of panels have.

![image](https://user-images.githubusercontent.com/333268/140361954-f3185a06-73ba-4290-9b09-0b5b9af3d4a2.png)

Is there any other way for closing/minimizing the panel?

Thanks.

**Comment by KlausEverWalkingDev** [comment]
Have you tried the View in the Main Menu?

**Reply by jansorg** [reply]
This is not a tool window, but a split editor window. Is it possible that you turned off the visibility of editor tabs?
![Screenshot_20211104_171421](https://user-images.githubusercontent.com/11473063/140377184-6c8772fd-3500-40e0-823c-e077fbf6e279.png)

**Comment by duxtinto** [comment, marked as answer]
Hi @KlausEverWalkingDev and @jansorg !

Thank you very much for you quick answer.

As pointed by @jansorg , the issue was related to having the `tab placement` setting set to `none` value

![image](https://user-images.githubusercontent.com/333268/140495950-b598c27a-7008-4886-b0dd-aad8c3b59874.png)

Thanks

**Reply by srsholmes** [reply]
Just fyi, if you want to keep the editor tabs set to hidden, you can click in the github copilot 'tab' and then close it by doing `cmd / ctrl W`, or whatever the shortcut you have set to close tabs is.
